src/NServiceBus.Core.Tests/Pipeline/Outgoing/OutgoingPublishContextTests.cs

```diff
@@ -7,30 +7,13 @@
     public class OutgoingPublishContextTests
     {
         [Test]
-        public void ShouldShallowCloneHeaders()
```

Comment:
yes it's still revlevant, but it's no longer this classes responsibility to clone the headers. Added this as a new unit test to the MessageOperations.Publish tests.